src/app/[lang]/layout.tsx

```diff
@@ -2,6 +2,10 @@ import { server } from "@/mocks/node";
 import type { Metadata } from "next";
 import { Geist, Geist_Mono } from "next/font/google";
 import "./globals.css";
```

Comment:
I guess this import should be updated
```suggestion
import "@/app/globals.css";
```